packages/ses/src/lockdown.js

```diff
@@ -385,7 +385,8 @@ export const repairIntrinsics = (options = {}) => {
     // must be the operation that modifies the intrinsics.
     tamedHarden(intrinsics);
 
-    // Harden Compartment constructor and evaluators in start Compartment
+    // Harden Symbol, Compartment constructor and evaluators in start Compartment
```

Comment:
Small nit. I think we should visually separate the hardening of `Symbol` from the evaluators.

My reasoning is that `Symbol` inside a compartment is shared between all compartments, unlike evaluators which all have their own distinct ones.

I'm also wondering if there are other "initial" only intrinsics that should similarly be hardened. `Error` comes to mind, and I'm wondering if it's already hardened, and if so, why `Symbol` is not handled similarly to `Error`.

Reply:
1. In that ctx, should `Compartment` be separated as well as `Symbol`? We don't consider it to be an evaluator right?
2. `Error` isn't hardened on the start compartment, before deciding on that might be useful to list those - is there a reference to the group of intrinsics that a Compartment gets that are new instances of their original instance on the start Compartment rather than identical instances? (e.g. `Function`, `eval`, `Error`, etc)

Reply:
`Compartment` is definitely an evaluator. `new Compartment().evaluate()` will eval code.

Regarding things that are only on the initial compartment's global, see `initialGlobalPropertyNames` in `permits.js`. `Symbol` is excluded there, for reasons I actually don't fully understand. The comment seem to indicate these intrinsics are somehow repaired, but I'm wondering why. Regardless, it's possible we want all these initial only intrinsics to be hardened during `hardenIntrinsics`. Paging @erights

Reply:
What I don't follow is how adding something to `initialGlobalPropertyNames` ends up having any kind of implication besides it being assigned to the global object.

Reply:
Oh there is no `%InitialSymbol%` defined anywhere. Why don't we capture the original `Symbol` as `%InitialSymbol%` instead of this weird dance? Then all these initial intrinsics can be given the same treatment.

Reply:
Because then we'd need to list all its properties that we do not want removed, which we assume we cannot know ahead of time.

Or, if you have something in mind you think will work, try it. The PR should either be trivial or impossible ;)

Reply:
@mhofman fix d3163f722bfe8df507cbf86132c04d4af35545b5

Also, I noticed the list of evaluators we wish to harden is almost identical to `uniqueGlobalPropertyNames` in `permits.js` (except for `globalThis`), perhaps we should use it instead of naming all three by name?

Reply:
I have to admit I don't fully grasp the purpose of `uniqueGlobalPropertyNames` as it doesn't seem used anywhere. I suppose it's similar to `initialGlobalPropertyNames`, which is from permits, but seem only used to install some properties onto the initial compartment's global object.

I'd say for now, we can keep the explicit hardening of the evaluators.